app/bundles/PageBundle/Entity/HitRepository.php

```diff
@@ -273,6 +273,10 @@ public function getLatestHit($options)
         }
         $result = $sq->execute()->fetch();
 
+        if (false == $result) {
+            return $result;
+        }
+
         return new \DateTime($result['latest_hit'], new \DateTimeZone('UTC'));
```

Comment:
I would suggest more explicit return statement using null when hit is not found.
```suggestion
return $result ? new \DateTime($result['latest_hit'], new \DateTimeZone('UTC')) : null;
```